Realm/Realm/Native/NativeCommon.cs

```diff
@@ -83,7 +83,7 @@ public static unsafe void Initialize()
                         var assemblyLocation = Path.GetDirectoryName((string)assemblyLocationPI.GetValue(typeof(NativeCommon).GetTypeInfo().Assembly));
                         var architecture = InteropConfig.Is64BitProcess ? "x64" : "x86";
                         var path = Path.Combine(assemblyLocation, "lib", "win32", architecture) + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH");
-                        Environment.SetEnvironmentVariable("PATH", path);
+                         Environment.SetEnvironmentVariable("PATH", path);  // hack we can do for Windows but not UWP
```

Comment:
Does control even enter this branch? I'd assume it gets filtered out for UWP. Also, there's a leading space added.